### ISSUE TITLE
Lock-in design system moving forward

### ISSUE THREAD

**Issue by 0xTranqui** [1ifeworld/river#83]
tldr is that we currently do not have an end-to-end method for styling at the code implementation level. I just ran into a blocker where when trying to implement code, we dont have the new figma colors addded into the globals.css sheet, meaning I am needing to inline variables using tailwind, which in my opinion is non-viable for this version of the build (production ready).

mainly using this issue as a convo entrypoint to get things finalized. whatever our collective knowledge is for best implementing a design system from figma -> globals.css -> impelmenting things in code, we need this done with high priority because it is blocking production-level implementation of the frontend. even if we arent in love with these decisions, they cannot be put off any longer, for risk of introducing errors + slowing down ability to refactor caused by lack of a coherant system

@salieflewis maybe you could try to tease out the "decisions" that need to be made from the code standpoint, and what you are either missing or still deliberating on that you need help resolving before making a final decision?

def down to lend my input on anything if questions remain, just lemme know.

cc @ioeylim for visibility

**Comment by ioeylim** [comment]
Can put together a starting-point design system tomorrow which includes:
1. Color variants (light mode to begin with) 
2. Font variants 
3. Miscellaneous (various widths, general spacing, etc...) 

This should take us to at least the 85% mark in terms of styling.

**Comment by salieflewis** [comment]
@ioeylim it would be helpful if you read through [this](https://ui.shadcn.com/docs/theming) which is how all the design system stuff is setup right now out of the box. I would like to augment it as little as possible while providing flexibility for what we need to be 1:1 with the designs. Happy to chat about this at any time tomorrow as well.

**Comment by ioeylim** [comment]
> @ioeylim it would be helpful if you read through [this](https://ui.shadcn.com/docs/theming) which is how all the design system stuff is setup right now out of the box. I would like to augment it as little as possible while providing flexibility for what we need to be 1:1 with the designs. Happy to chat about this at any time tomorrow as well.

Did a quick read through and seems pretty straight forward. Will let you know if i have any questions tomorrow

**Comment by salieflewis** [comment]
Resolved #90
